test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -378,6 +378,11 @@ using `
         $res.CompletionMatches[0].CompletionText | Should -BeExactly '${PSVersionTable}'
     }
 
+    It 'Should work for property assignment of enum type:' {
+        $res = TabExpansion2 -inputScript '$psstyle.Progress.View="Clas'
```

Comment:
Does it allow spaces around `=`, like `$psstyle.Progress.View = <tab>`?

Reply:
Yes, it uses the same triggers that the variable assignment completion does so I figure there's no need to add tests for all the different scenarios because the variable assignment tests already covers them.